### DISCUSSION TITLE
Able to make on-demand build servers

### DISCUSSION THREAD

**Discussion by andrasbacsai** [coollabsio/coolify#2287] [Feature Requests]
When you deploy, it quickly spin up a server (like on Hetzner), setup it, build the app and stops the server.

## Prereqs:

1. API connection to hosting providers
2. Automated Docker Registry authentication

**Comment by thansk** [comment]
Certainly a good idea. 

Who would maintain the integrations with different services?

**Comment by Schmedu** [comment]
Just thought about it. Definitely needed! 😬

**Comment by JasonLovesDoggo** [comment]
Or perhaps this could be offloaded to CICD platforms like github/lab actions? Deploy after image is built
